infra/src/database.ts

```diff
@@ -46,6 +57,205 @@ const dbInstance = new aws.rds.Instance(nm("app"), {
   finalSnapshotIdentifier: nm("app-db-final-snapshot"),
 });
 
+const encryptedDbInstance = new aws.rds.Instance("app-encrypted", {
```

Comment:
use the `nm` helpers so we see the environment name in the resource names in AWS.